core/src/test/java/org/apache/calcite/test/ReflectiveSchemaTest.java

```diff
@@ -63,9 +65,7 @@
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
do not use `.* `

Reply:
Apologies... I'll disable the IntelliJ automation

Reply:
Completed in #3634 .